Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.cpp

```diff
@@ -5,9 +5,13 @@
 
 #include <algorithm>
 
+#include <QObject>
```

Comment:
You probably shouldn't be referring to a specific UI framework in here (and unless I missed something, you don't even need it included).